Drift/Public/DriftAPI.h

```diff
@@ -298,6 +298,7 @@ enum class EDriftPresence : uint8
 UENUM(BlueprintType)
 enum class EDriftFriendType : uint8
 {
+    NotFriend,
```

Comment:
I would have to look up if this works well in BPs that already reference the old enum values, since they're now getting remapped +1 here.

Reply:
In case they don't get recompiled when a referenced type changes?  That would be a fairly big oops if that was the case imo...

Reply:
I'm pretty sure they're actually referenced by name, so re-ordering should be fine, and since everything is recompiled as needed we don't really need to worry about the ABI between components.